src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/RegisterAktivteter.tsx

```diff
@@ -26,17 +27,17 @@ const RegisterAktiviteter: React.FC<{
     const aktiviteter = grunnlag.aktivitet.aktiviteter;
     const hentetInformasjon = grunnlag.hentetInformasjon;
 
-    const opplysningerHentetTekst = `Hentet fra Arena: ${formaterNullableIsoDatoTid(hentetInformasjon.tidspunktHentet)}`;
-
-    if (aktiviteter.length === 0) {
+    if (aktiviteter.length > 0) {
```

Comment:
Burde ikke denne fortsatt være: `if (aktiviteter.length === 0) {` ?

Reply:
Jo, takk. Endret for å få testet en ting 🐤